client-data/tools/text/text.js

```diff
@@ -133,6 +167,13 @@
 				if (textField === null) {
 					console.error("Text: Hmmm... I received text that belongs to an unknown text field");
 					return false;
+				} else {
+					/*if(Tools.useLayers){
+						if(textField.getAttribute("class") !== "layer"+Tools.layer){
+							textField.setAttribute("class","layer-"+Tools.layer);
+							Tools.group.appendChild(textField);
+						}
+					}*/
```

Comment:
can we remove this ?

Reply:
I will reintroduce it when I implement the masking eraser. Having it already present will make that step faster, although I can of course just commit it again

Reply:
Yes, we can re-add that later if we introduce layers, but currently, we shouldn't have commented-out code.